app/gui/src/dashboard/components/styled/withFocusScope.tsx

```diff
@@ -12,7 +12,7 @@ import * as aria from '#/components/aria'
  * used in the component.
  */
 // This is not a React component, even though it contains JSX.
```

Comment:
this comment is associated with the `no-restricted-syntax` lint and should be removed if the lint no longer applies.
(there are 5 matches here, but it's safest to do a find all to find every occurrence in the codebase)